aliasing/api/context.py

```diff
@@ -206,6 +206,9 @@ def __init__(self, author):
         self._id = author.id
         self._discriminator = author.discriminator
         self._display_name = author.display_name
+        self._roles = []
+        if isinstance(author, disnake.Member):
+            self._roles = [AliasRole(role) for role in author.roles]
```

Comment:
Maybe use a generator here so the AliasRoles are generated upon request at not at the start of every alias
```suggestion
            self._roles = (AliasRole(role) for role in author.roles)
```

Reply:
Ignore this, forgot generators are single use